enterprise-suite/values.yaml

```diff
@@ -9,11 +9,17 @@ consoleUIConfig:
 # Update these versions for a release.
 #
 # lightbend-docker-commercial-registry.bintray.io/enterprise-suite/es-console
-esConsoleVersion: v1.4.9
+# Github repo: https://github.com/lightbend/console-gui/
```

Comment:
This file is open to everyone so I'm not sure we want to add references to private repos here.

Ditto for other comments here.

Reply:
Remove the URLs to the repositories. I've kept the names, though.